templates/content-single-ms_directory.php

```diff
@@ -670,10 +670,9 @@ function manager_industry( $manager_industry ) {
 										<?php
 										if ( isset( $url ) ) {
 											?>
-											<a href="<?= esc_url( add_query_arg( array( 'directory_name' => get_the_title(), 'directory_url' => get_the_permalink() ), $url ) ); // @codingStandardsIgnoreLine ?>" class="AffiliateManagerCard__image--showProfile" title="<?= esc_html( $url_title ); ?>">
-									<?= $picture; // @codingStandardsIgnoreLine ?>
-									<span><?php _e( 'Show profile', 'ms' ); ?></span>
-								</a>
+                                            <a href="<?= esc_url( add_query_arg( array( 'directory_name' => get_the_title(), 'directory_url' => get_the_permalink() ), $url ) ); // @codingStandardsIgnoreLine ?>" class="AffiliateManagerCard__image" title="<?= esc_html( $url_title ); ?>">
```

Comment:
Please fix intendation of <a href part (too many spaces, use less intendation using tabs)

Reply:
fixed